src/common/regions.ts

```diff
@@ -1,17 +1,18 @@
 import { createContext, useContext } from 'react';
-import { chain, fromPairs } from 'lodash';
+import { chain, deburr, words, fromPairs, sum } from 'lodash';
 import urlJoin from 'url-join';
 import { assert } from 'common/utils';
 import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset_01_02_2020.json';
 import countyAdjacencyMsa from './data/county_adjacency_msa.json';
+import msaDefinitions from './data/msa-data.json';
 const { state_dataset, state_county_map_dataset } = US_STATE_DATASET;
 
 export type FipsCode = string;
 
 export enum RegionType {
   COUNTY = 'county',
   STATE = 'state',
-  CBSA = 'CBSA',
+  MSA = 'MSA',
```

Comment:
I might have mentioned this in a different review but is it clear that "MSA" is the right term for us to use throughout our code as opposed to "CBSA" I guess?  Or maybe we need some other term if it's not strictly an MSA or CBSA (MetroArea?)?

Maybe you or somebody else just needs to explain to me exactly what we mean by "MSA" and this will all make sense though. 😁

Reply:
I think part of the confusion here is that we will be showing a subset of CBSAs on the site, they might be the same as all of the MSAs.  
I don't have a super strong preference, but would lean towards referring to them as CBSAs even though its a subset - for example in the future if we wanted to include micropolitan areas, I don't think the data structure would look any different

From a [good blog post](https://www.inc.com/encyclopedia/metropolitan-statistical-area-msa.html#:~:text=Core%2DBased%20Statistical%20Area,core%20and%20the%20outlying%20areas.&text=The%20CBSA%20is%20a%20generic,difference%20being%20core%20population%20size.)
> A CBSA is one or more counties with an urbanized cluster of at least 10,000 people. The area as a whole is defined by the interaction between the core and the outlying areas. This interaction, measured by commuting, means that at least 25 percent of people in outlying areas are working in the core. The CBSA is a generic definition of MICROs and MSAs, the difference being core population size.

Reply:
Let's keep it as MSA for now, we can always define CBSA as a union type if we ever add µSA to the site.